### PR TITLE
Update kind version and engineerd/setup-kind action commit

### PR DESCRIPTION
**Pull request by cpretzer**
* Update inteegration_tests.yml to determine whether kind is too outdated to run properly 
* 
Signed-off-by: Charles Pretzer <charles@buoyant.io>

### CODE REVIEW

**Review by mateiidavid — APPROVED** [review]

File: iptables/iptables.go [hunk 1]
```diff
@@ -125,8 +125,6 @@ func addOutgoingTrafficRules(commands []*exec.Cmd, firewallConfiguration Firewal
 	// Ignore traffic from the proxy
 	if firewallConfiguration.ProxyUID > 0 {
 		log.Printf("Ignoring uid %d\n", firewallConfiguration.ProxyUID)
-		// Redirect calls originating from the proxy destined for an app container e.g. app -> proxy(outbound) -> proxy(inbound) -> app
-		commands = append(commands, makeRedirectChainForOutgoingTraffic(outputChainName, redirectChainName, firewallConfiguration.ProxyUID, "redirect-non-loopback-local-traffic"))
```
Comment (olix0r) [inline iptables/iptables.go:129]: we should probably back-out these changes and make them in a separate PR?

**Comment by olix0r** [comment]
Merged into #45
